### PR TITLE
Add base URI metadata fetches

### PR DESCRIPTION
**Pull request by fhildeb**

### CODE REVIEW

**Comment by CJ42** [comment]
Added review comment

File: digital-assets/base-uri-metadata.ts [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  const jsonMetadata = await response.json();
+  console.log('Metadata Contents: ', jsonMetadata);
+}
+
+// Get the  global token ID format of the asset
+const tokenIdFormat = await getTokenIdFormat();
+console.log('Token ID Format: ', tokenIdFormat);
+
+if (tokenIdFormat !== null) {
+  fetchBaseURI(byte32TokenId, tokenIdFormat);
```
Comment (b00ste) [inline digital-assets/base-uri-metadata.ts:145]: `fetchBaseURI` name is confusing, shouldn't it be `fetchMetadata`?

**Comment by fhildeb** [comment]
@CJ42, could you have a look at the build issues, please? They are showing up for `.sol` files and should not be related to this file.

**Comment by Hugoo** [comment]
@fhildeb @b00ste the CI is failing. Can you please take a look and merge this?

**Review by CJ42 — APPROVED** [review]

**Pull request merged**